hackIDE_project/settings.py

```diff
@@ -23,7 +23,8 @@
 SECRET_KEY = '+h*i@$52+w(_e#etvzgnkjq!q0ajz1qpgs-y9%89x4w3nlct=m'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = (os.environ.get('HACKIDE_DEBUG') != None)
+DEBUG = True
```

Comment:
I was getting bad request 400. But when i set debug = Truee, it works.

On Aug 11, 2016 9:43 PM, "Sahil Dua" <notifications@github.com> wrote:

> In hackIDE_project/settings.py
> <https://github.com/sahildua2305/hackIDE/pull/36#discussion_r74453277>:
>
> > @@ -23,7 +23,8 @@
> >  SECRET_KEY = '+h*i@$52+w(_e#etvzgnkjq!q0ajz1qpgs-y9%89x4w3nlct=m'
> >
> >  # SECURITY WARNING: don't run with debug turned on in production!
> > -DEBUG = (os.environ.get('HACKIDE_DEBUG') != None)
> > +DEBUG = True
>
> What's the issue with this? We need to fix it if it doesn't work with DEBUG
> = False. In production, it should always have DEBUG = False
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sahildua2305/hackIDE/pull/36/files/931cff6349d93bbff04776aeadc59a2f2c215ab7#r74453277>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALtV3jE4lj_v_zfoGXIDXF_vLUCIy2Iqks5qe0oUgaJpZM4Jh2iO>
> .
>